subsys/bluetooth/mesh/adv_ext.c

```diff
@@ -173,6 +192,13 @@ static int adv_start(struct bt_mesh_ext_adv *adv,
 
 	adv->timestamp = k_uptime_get();
 
+	if (adv->tag & BT_MESH_FRIEND_ADV) {
```

Comment:
It is probably not needed to set zero randomness here as it is constant configuration IIRC. Could you check without this code?

Reply:
Yep, you're right